apps/console/src/features/email-management/pages/email-customization.tsx

```diff
@@ -352,11 +360,22 @@ const EmailCustomizationPage: FunctionComponent<EmailCustomizationPageInterface>
                         >
                             { t("extensions:common.learnMore") }
                         </DocumentationLink>
-                    </>
-                ) }
+                    </> )
+                }
                 titleTextAlign="left"
                 bottomMargin={ false }
                 data-componentid={ componentId }
+                action={
+                    featureConfig.emailProviders?.enabled &&
+                    (
+                        <SecondaryButton
+                            onClick={ goToEmailProvider }
+                            data-componentId="email-provider-configure-button"
+                        >
+                            { "Configure " + t("extensions:develop.sidePanel.emailProvider") }
```

Comment:
Let's add this as a single i18n placeholder.

Reply:
fixed https://github.com/wso2/identity-apps/pull/4202/commits/8ab65827fa45aca316b7682aeeb2af8236bfaaa8